openshift/app.dc.yaml

```diff
@@ -133,6 +153,42 @@ objects:
                     name: cdogs-server-config
                 - configMapRef:
                     name: cdogs-common-service-config
+            # Fluent-bit sidecar
+            - image: fluent/fluent-bit:1.5
+              imagePullPolicy: Always
+              name: fluent-bit
+              resources:
+                requests:
+                  cpu: "${SIDECAR_CPU_REQUEST}"
+                  memory: "${SIDECAR_MEMORY_REQUEST}"
+                limits:
+                  cpu: "${SIDECAR_CPU_LIMIT}"
+                  memory: "${SIDECAR_MEMORY_LIMIT}"
+                readinessProbe:
+                  httpGet:
+                    path: /
+                    port: 2020
+                livenessProbe:
+                  httpGet:
+                    path: /
+                    port: 2020
+              volumeMounts:
+              # mount log file location
+              - name: log-storage
+                mountPath: /var/log
+              # mount fluent-bit config file
+              - name: fluent-bit-config
+                mountPath: /fluent-bit/etc/
+              # TLS cert for connecting to fluentd (enable for TLS)
+              # - mountPath: /fluent-bit/ssl
+              #   name: fluent-bit-tls
+              ports:
+                - containerPort: 2020
+                  protocol: TCP
+                  name: metrics
+                - containerPort: 80
+                  name: http-plugin
+                  protocol: TCP
```

Comment:
Let's keep ordering consistent and in alphabetical order whenever possible -> containerPort, name, then protocol